ui/narrative/methods/annotate_contigs/display.yaml

```diff
@@ -19,7 +19,7 @@ suggestions:
             []
     methods:
         related:
-            []
+            [annotate_contigset, annotate_domains_in_a_genome, annotate_plant_transcripts, reannotate_microbial_genome]
```

Comment:
This broke a while back (see https://kbase-jira.atlassian.net/browse/KBASE-3778) -- these related/next "methods" (apps) used to be shown in the app cells, and they were quite helpful.

When I started adding these fields to display.yamls, I didn't realize that was still broken--I thought the apps just didn't have next apps set in their configs. So I added them to this yaml, and one or two others, before realizing that in fact this is still broken.

It's probably harmless to have them here; if the functionality of showing next/related apps is fixed at some point, then these will show up.